app/templates/events/view/tickets/add-order.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="row" style="padding-top:10px">
+  <h2 class="ui header">{{t 'Add Order'}}</h2>
+</div>
+<table class="ui very basic table">
+  <thead>
+    <tr>
+      <th>{{t 'Ticket Type'}}</th>
+      <th class="right aligned">{{t 'Price '}}(US$)</th>
+      <th class="right aligned">{{t 'Quantity'}}</th>
+      <th class="right aligned">{{t 'Item Total'}}</th>
+    </tr>
+    <tr>
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type='number' min='0'}}</td>
```

Comment:
Update Total & grand total when we change quantity of ticket. check how this was implemented in `ticket-list` component in the public route.